components/cert/CertificateTemplate4.js

```diff
@@ -0,0 +1,658 @@
+                                        d="M311.75,91.02c.54,.31,.58,1.05,.12,2.24-.66,.87-1.37,1.67-2.12,2.43s-1.57,1.42-2.46,2.02c-.07,0-.1,.02-.1,.07-.59,.41-1.22,.76-1.88,1.06-5.45,1.96-8.23,.81-8.34-3.45-2.21,1.91-3.69,3.09-4.44,3.52-1.8,1.07-3.75,1.55-5.84,1.44-1.55-.09-2.6-.56-3.16-1.42-.56-.85-.51-2.01,.15-3.47,1-2.3,2.43-4.35,4.27-6.15,2.16-2,4.57-3.61,7.21-4.82,1.41-.71,2.89-.98,4.44-.82,1.55,.09,2.48,.97,2.8,2.63,.14,.5,.01,.95-.38,1.33-.5,.46-1.08,.56-1.74,.31-.05,0-.11-.01-.19-.03-.08-.02-.14-.06-.19-.1-.66-.91-1.33-1.29-2-1.14-.67,.15-1.38,.45-2.14,.91-3.51,2.05-6.12,4.91-7.83,8.58-.09,.16-.15,.34-.17,.53-.02,.19,.03,.35,.17,.46,.41,.14,.87,.11,1.37-.08,.5-.19,1-.48,1.5-.86,.5-.38,.92-.69,1.26-.94l.07-.1c1.66-1.39,3.29-2.81,4.89-4.27,.71-.75,1.58-1.17,2.63-1.26,1.5,0,2.13,.67,1.88,2.02-.14,.5-.29,1-.44,1.5-.59,2.17-.62,3.44-.07,3.81,.55,.38,1.38,.38,2.51,0,1.13-.38,2.26-1.09,3.38-2.15,1.13-1.06,2.08-2.03,2.85-2.91,.77-.88,1.43-1.16,1.96-.85Z" />
+                                    <path class="cls-3"
+                                        d="M335.32,93.88c-1.28,1.21-2.63,2.22-4.07,3.04s-3.1,1.64-4.99,2.46c-1.25,.55-2.59,.73-4,.55-1.41-.18-2.12-1.04-2.12-2.56s.35-3.04,1.06-4.55c.71-1.55,.86-2.35,.46-2.39-.4-.04-1.23,.33-2.48,1.13-3.26,2.14-6.29,4.59-9.09,7.35-.75,.91-1.71,1.41-2.87,1.5-.59,0-1.05-.19-1.38-.56-.33-.38-.37-.78-.12-1.21,2.01-3.71,4.03-7.38,6.08-11.01,.46-.75,1.14-1.22,2.05-1.4,.39-.07,.75,.05,1.08,.36,.33,.31,.5,.66,.5,1.04-.05,.37-.19,.8-.44,1.3-.11,.3-.22,.58-.33,.85-.1,.27-.14,.5-.1,.68,.03,.18,.18,.24,.43,.17,.34-.18,.68-.38,1.03-.58,.87-.5,1.7-.98,2.5-1.44,.71-.41,1.45-.77,2.22-1.09,.77-.32,1.58-.54,2.43-.65,1.37-.09,2.33,.22,2.91,.92,.57,.71,.64,1.69,.21,2.94-.46,1.14-.93,2.27-1.44,3.38-.62,1.05-.72,1.85-.31,2.39s2.31-.08,5.71-1.88l2.56-1.5c2.42-1.55,3.71-2.13,3.9-1.74,.36,.46-.09,1.29-1.37,2.5Z" />
+                                </g>
+                            </g>
+                            <line class="cls-1" y1="175.95" x2="460.4" y2="175.95" />
+                        </g>
+                    </g>
+                </svg>
+
+                <svg xmlns="http://www.w3.org/2000/svg">
+                    <style>
+                        @font-face {
+                            font - family: 'Product Sans Regular';
+                        src: url('ProductSans-Regular.ttf') format('truetype');
+            }
+
+                        text {
+                            font - family: 'Product Sans Regular';
+            }
+                    </style>
+                    <text x="260" y="670" fill="#4285f4" fontSize="37px" className="text__cls">
+                        Certificate ID: 001
+                    </text>
+                </svg>
+                <text x="560" y="670" fill="#4285f4" fontWeight="500" fontFamily="Roboto" fontSize="37px" className="text__cls">
+                    Issue Date: ${issueDate}
+                </text>
+            </svg>
```

Comment:
What is the purpose of issueDate? Do you really need to use two variables for the date? why?